dotCMS/src/main/java/com/dotcms/aspects/interceptors/CloseDBIfOpenedMethodInterceptor.java

```diff
@@ -2,34 +2,35 @@
 
 import com.dotcms.aspects.DelegateMethodInvocation;
 import com.dotcms.aspects.MethodInterceptor;
-import com.dotcms.business.CloseDB;
+import com.dotcms.business.CloseDBIfOpened;
 import com.dotmarketing.db.DbConnectionFactory;
 
 import static com.dotcms.util.AnnotationUtils.getMethodAnnotation;
 
 /**
- * Method handler for the {@link CloseDB} annotation aspect
+ * Method handler for the {@link CloseDBIfOpened} annotation aspect
  * @author jsanca
  */
-public class CloseDBMethodInterceptor implements MethodInterceptor<Object> {
+public class CloseDBIfOpenedMethodInterceptor implements MethodInterceptor<Object> {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)